.envrc

```diff
@@ -0,0 +1,125 @@
+set -e
+layout node
+
+# nvm based on https://github.com/steve-ross/direnv-helpers
+
+__prompt_install_nvm(){
+  _log warn "Couldn't find nvm (node version manager)..."
+  read -p "Should I install it? " -n 1 -r
+  echo    # (optional) move to a new line
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    _log info "Installing NVM"
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+    __source_nvm # make sure nvm is sourced
+  else
+    log_error "Install nvm first and make sure it is in your path and try again"
+    _log warn "To install NVM visit https://github.com/creationix/nvm#installation"
+    exit
+  fi
+}
+
+__npm_install_and_layout(){
+  if [ ! -d ./node_modules ]; then
+    # no node modules... run npm install
+    npm install
```

Comment:
Isn't this sorta what we don't wanna do? Shouldn't we be `yarn install`ing?

Also kinda weird IMO that we would `npm install` just by `cd` into a directory.